newIDE/app/src/ResourcesList/index.js

```diff
@@ -291,12 +291,23 @@ export default class ResourcesList extends React.Component<Props, State> {
       { type: 'separator' },
       {
         label: i18n._(t`Remove unused...`),
-        submenu: allResourceKindsAndMetadata.map(({ displayName, kind }) => ({
-          label: i18n._(displayName),
-          click: () => {
-            this.props.onRemoveUnusedResources(kind);
-          },
-        })),
+        submenu: allResourceKindsAndMetadata
+          .map(({ displayName, kind }) => ({
+            label: i18n._(displayName),
+            click: () => {
+              this.props.onRemoveUnusedResources(kind);
+            },
+          }))
+          .concat([
+            {
+              label: i18n._('All resources'),
```

Comment:
You need to mark strings to translate with the t macro, otherwise it won't work:
```suggestion
              label: i18n._(t`resources (any kind)`),
```